changelog.d/442.improvements

```diff
@@ -0,0 +1 @@
+[Export e2ee keys] Update file name: element-megolm-export-$userId-$timestamp.txt
```

Comment:
Can you find a better changelog message? Not sure it would be clear for the users.
Something like `[Export e2ee keys] Update the default file name` WDYT